rudder-core/src/main/scala/com/normation/rudder/services/policies/write/BuildBundleSequence.scala

```diff
@@ -118,6 +120,10 @@ object BuildBundleSequence {
   // The rudder id is ruleid@@directiveid@@0
   final case class ReportId(value: String)
```

Comment:
Isn't reportId the same thing that BundleContext ?

Reply:
The identifier produced at the end is the report id, But i wanted to build it agent side by sending ruleId and directiveid separately so we can use then more easily in the technique ('maybe add debug / verbose info ...